src/test/java/com/example/pressarticle/article/ArticleServiceTest.java

```diff
@@ -0,0 +1,143 @@
+package com.example.pressarticle.article;
+
+import com.example.pressarticle.article.model.Article;
+import com.example.pressarticle.article.repository.ArticleRepository;
+import com.example.pressarticle.article.service.ArticleService;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.Pageable;
+
+import java.sql.Timestamp;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@ExtendWith(MockitoExtension.class)
+class ArticleServiceTest {
+
+    @Mock
+    ArticleRepository articleRepository;
+
+    @InjectMocks
+    ArticleService articleService;
+
+    @DisplayName("Should return sort Articles")
+    @Test
+    void sortAllTransfer() {
+        //given
+        List<Article> list = new ArrayList<>();
+        list.add(new Article(1L, "Machines replace human","Machines", LocalDate.of(2022,9,17),
+                "World", "Allan Balkier", new Timestamp(100, 10, 11, 0, 0, 0, 0)));
+        list.add(new Article(2L, "Robots are a system computer","Robots", LocalDate.of(1022,8,11),
+                "World", "Allan Balkier", new Timestamp(100, 10, 11, 0, 0, 0, 0)));
+
+        list.sort(Comparator.comparing(Article::getDataPublication).reversed());
+
+        Mockito
+                .when(
+                        articleRepository.findAll(Mockito.any(Pageable.class))
+                )
+                .thenReturn(new PageImpl<>(list));
+
+        //when
+        List<Article> articleList =  articleService
+                .sortAllArticleTransfer(0, 20, "data_Publication");
+
+        //then
+        assertEquals(list.get(0), articleList.get(0));
+    }
+    @DisplayName("Should return Articles for id")
+    @Test
+    void articleIdTransfer() {
+        //given
+        List<Article> article = new ArrayList<>();
+        article.add(new Article(1L, "NewWorld", "NewTitle",LocalDate.of(2022,9,17),
+                "World", "Allan Balkier", new Timestamp(100, 10, 11, 0, 0, 0, 0)));
+
+        Mockito.when(articleRepository.findArticleById(Mockito.anyLong())).thenReturn(article);
```

Comment:
Zbyt łagodnie zamockowałeś tę metodę - sprawdzasz tylko czy ktoś ją zawoła z dowolnym id - a zdecydowanie powinniśmy kontrolować czy to jest id przekazane do serwisu